inc/saihostif.h

```diff
@@ -409,6 +409,13 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_MPLS_ROUTER_ALERT_LABEL = 0x00007004,
 
+    /**
+     * @brief MPLS packet with label trapped to CPU
+     * This is the MPLS packet with incoming label map entry
+     * programmed to trap it to CPU
+     */
+    SAI_HOSTIF_TRAP_TYPE_MPLS_INSEG_ENTRY_TO_CPU = 0x00007005,
```

Comment:
Just like above messages, can you make the macro more explicit. I guess here you meant the trap for 'TTL expiry'.

Reply:
Didn't get you. This trap is for those packets which hit inseg entry with packet forward action is to CPU